sdk/translation/azure-ai-translation-document/samples/async_samples/sample_synchronous_document_translation_async.py

```diff
@@ -0,0 +1,66 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+"""
+FILE: synchronous_document_translation_async.py
+
+DESCRIPTION:
+    This sample demonstrates how to invoke synchronous document translation operations.
+
+USAGE:
+    python synchronous_document_translation_async.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_DOCUMENT_TRANSLATION_ENDPOINT - the endpoint to your Document Translation resource.
+    2) AZURE_DOCUMENT_TRANSLATION_KEY - your Document Translation API key.
+"""
+
+import asyncio, os
+from azure.core.credentials import AzureKeyCredential
+from azure.ai.translation.document.aio import SingleDocumentTranslationClient
+from azure.ai.translation.document.models import DocumentTranslateContent
+from azure.ai.translation.document._vendor import FileType
```

Comment:
Don't import from _vendor. If this isn't exposed in azure.core then redefine the type in this file

Reply:
Updated the import